sources/Common/DYNSimulationResult.cpp

```diff
@@ -80,6 +80,16 @@ SimulationResult::setStatus(status_t status) {
   status_ = status;
 }
 
+void
+SimulationResult::setCriticalTimeMessageError(const std::string& messageCriticalTimeError) {
```

Comment:
Please update the name of this method (set/getSimulationMessageError) and of the parameter